scripts/braid-fixture.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+set -eox pipefail
+
+repo=git@github.com:seek-oss/braid-design-system.git
```

Comment:
This script selectively checks out the `braid-design-system` package from the monorepo.